app/containers/MessageBox/EmojiSearchbar.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from 'react';
+import { View, Text, Pressable, TextInput, FlatList } from 'react-native';
+
+import { FormTextInput } from '../TextInput/FormTextInput';
+import { useTheme } from '../../theme';
+import I18n from '../../i18n';
+import { CustomIcon } from '../CustomIcon';
+import { IEmoji } from '../../definitions';
+import shortnameToUnicode from '../../lib/methods/helpers/shortnameToUnicode';
+import CustomEmoji from '../EmojiPicker/CustomEmoji';
+import styles from './styles';
+import useFrequentlyUsedEmoji from '../EmojiPicker/frequentlyUsedEmojis';
+
+const BUTTON_HIT_SLOP = { top: 15, right: 15, bottom: 15, left: 15 };
+const EMOJI_SIZE = 30;
```

Comment:
40

Reply:
40 seemed too large on android. I attached a screenshot but probably missed adding the comment. 
<img width="359" alt="Screenshot 2022-09-01 at 10 46 27 PM" src="https://user-images.githubusercontent.com/77742477/187974000-ff13c197-825a-40fd-9d55-172f0f700be7.png">

I'll do it if you think it looks fine.

Reply:
It's 36 on Figma

Reply:
32 looks good to me.